easybuild/easyconfigs/g/GEOS/GEOS-3.9.3-GCC-11.3.0.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = 'ConfigureMake'
+
+name = 'GEOS'
+version = '3.9.3'
```

Comment:
GEOS v3.10.3 is also available: https://libgeos.org/usage/download/

Do we want to bump the version or is there a reason for v3.9.3?

Reply:
d0a2215